ted_sws/notice_transformer/adapters/rml_mapper.py

```diff
@@ -0,0 +1,48 @@
+import abc
+import pathlib
+import subprocess
+
+class RMLMapperABC(abc.ABC):
+    """
+        This class is a general interface of an adapter for rml-mapper.
+    """
+
+    @abc.abstractmethod
+    def execute(self, package_path: pathlib.Path) -> str:
```

Comment:
I think we want to know a package_path, but also, the input XML data. 
Don't we want to know the path to the JAR file, as well? 

We can then copy the input data somewhere, such as a temp folder (inside the package), so that, the transformation rules are relative to the input data. I think this might be easier. But better ways may exist.